src/test/java/models/CustomerModel.java

```diff
@@ -0,0 +1,43 @@
+package models;
+
+import com.github.javafaker.Faker;
+import lombok.*;
+
+import java.util.Locale;
+
+@Getter
+@Setter
+
+public class CustomerModel {
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String address;
+    private String city;
+    private String state;
+    private String zipCode;
+    private final String countryDefault = "United Kingdom"; // You can set the default country here
+    private String login;
+    private String password;
+
+    Faker fakerEN = Faker.instance(new Locale("en-gb"));
+
+    public CustomerModel() {
```

Comment:
This will be obsolete when you will use correct decorators.

Reply:
Ok. Done.